### PR TITLE
docs: Updates for v9.1.1 and re-arranging roadmap docs

### PR DESCRIPTION
**Pull request by mmulji-ic**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [x] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [x] Added `!` to the type prefix if API or client breaking change
* [x] Targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
* [ ] Provided a link to the relevant issue or specification
* [x] Followed the guidelines for [building SDK modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules)
* [x] Included the necessary unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
* [ ] Added a changelog entry to `CHANGELOG.md`
* [x] Included comments for [documenting Go code](https://blog.golang.org/godoc)
* [x] Updated the relevant documentation or specification
* [x] Reviewed "Files changed" and left comments if necessary 
* [x] Confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] confirmed all author checklist items have been addressed 
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage

### CODE REVIEW

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 1]
```diff
@@ -1,66 +1,28 @@
 # The Cosmos Hub Roadmap
 
-This Cosmos Hub roadmap serves as a reference for the current planned features of upcoming releases, as well as providing a record of past releases.
+This Cosmos Hub roadmap serves as a reference for the current planned features of upcoming releases. For past releases, please see the following [document](./README.md).
```
Comment (mmulji-ic) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md:3]: @jtremback ~~pulled this content from the January blog. Can add the upcoming releases that we have planned (10 & 11)~~

File: docs/roadmap/README.md [hunk 2]
```diff
@@ -1,17 +1,81 @@
 <!--
 order: false
 parent:
-  title: Roadmap
+  title: Previous Releases
   order: 7
 -->
 
-# Roadmap
+# Previous Releases
```
Comment (mmulji-ic) [inline docs/roadmap/README.md:8]: @jtremback moved the release changelog to here.

File: docs/roadmap/README.md [hunk 3]
```diff
@@ -1,17 +1,81 @@
...
+- IBC Packet Forward Middleware v3.1.1
+- IBC Msg Whitelist to skip MinFee in CheckTX
+- Global Fee Module
+  - Allows denoms and min-fees to be governance parameters so gas can be paid in various denoms.
+  - Visible on [tgrade](https://github.com/confio/tgrade/tree/main/x/globalfee) already and enabled in [ante.go](https://github.com/confio/tgrade/blob/main/app/ante.go#L72-L92)
+
+### v9-Lambda Upgrade (expected Q1 2023)
+
+- Gaia v9.0.x
+- Cosmos SDK v0.45
```
Comment (sainoe) [inline docs/roadmap/README.md]: Maybe it's worth specifying the SDK ICS fork that was used i.e. v0.45.13-ics

**Review by sainoe — APPROVED** [review]

File: docs/roadmap/cosmos-hub-roadmap-2.0.md (hunk 1 above)
Comment (mmulji-ic) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md:3]: Now use the living roadmap google doc link that wasn't available before.

**Review by jtremback — APPROVED** [review]
LGTM- I just wonder what will happen to external links. Is there any possibility of doing redirects?

**Comment by mmulji-ic** [comment]
> LGTM- I just wonder what will happen to external links. Is there any possibility of doing redirects?

This PR is just shuffling content around and not deleting anything, so external links still should be preserved.

**Pull request merged**